docs/containers-policy.json.5.md

```diff
@@ -253,6 +255,8 @@ selectively allow individual transports and scopes as desired.
             /* Similarly, allow installing the “official” busybox images.  Note how the fully expanded
                form, with the explicit /library/, must be used. */
             "docker.io/library/busybox": [{"type": "insecureAcceptAnything"}]
+            /* Allow installing images from all subdomains of access.redhat.com */
+            "*.access.redhat.com": [{"type": "insecureAcceptAnything"}]
```

Comment:
It’s not ideal to recommend insecure use of signed RH images. Something like `*.temporary-project.example.com`? Or maybe show how to require RH signatures for all of `*.redhat.com` (but that could detract from the `*.` point).